### PR TITLE
Improve Evals Pages

### PR DESCRIPTION
**Pull request by mbillow**
- Fix dropdown display login on intro evals page.
- Actually display pending results instead of 65535
- Match freshmen project pending status to pending packet.
- Fix adding issues when dropdown is hidden.

screen shot 2016-09-06 at 16 52 17

### CODE REVIEW

**Comment by liam-middlebrook** [comment]
Why remove the other notes section?

**Comment by liam-middlebrook** [comment]
Why remove the other notes section?

**Comment by liam-middlebrook** [comment]
Why remove the other notes section?

**Comment by liam-middlebrook** [comment]
Why remove the other notes section?

**Comment by liam-middlebrook** [comment]
Why remove the other notes section?

**Comment by liam-middlebrook** [comment]
Why remove the other notes section?

**Comment by mbillow** [comment]
There is no form where data can be entered into that field and I don't see any use for it. Correct me if I am wrong.

**Comment by mbillow** [comment]
There is no form where data can be entered into that field and I don't see any use for it. Correct me if I am wrong.

**Comment by liam-middlebrook** [comment]
`/intro_evals_form` totally exists. You can't see it because you aren't in intromembers.

**Comment by mbillow** [comment]
Fixed that... whoops!

**Comment by liam-middlebrook** [comment]
I really like the "Internal Account" marking.

File: conditional/templates/intro_evals.html [hunk 1]
```diff
@@ -20,16 +20,21 @@
...
+                                <h6 class="eval-uid">{{ m['uid'] }}</h6>
+                            {% endif %}
+
                     </div>
                     <!---->
                     <div class="col-sm-5 col-md-6 col-lg-5 vcenter">
                         <div class="row">
                             <div class="text-center">
-                                {% if m['signatures_missed'] > 0 %}
+                                {% if m['signatures_missed'] == 65535 %}
```
Comment (stevenmirabito) [inline conditional/templates/intro_evals.html]: I don't really like using "magic" numbers. Can we use an enum or just a nil value here to represent this instead?

**Comment by stevenmirabito** [comment]
Other than my one comment, this looks great! 👍

**Pull request merged**